### PR TITLE
CICD for client added via var.conf

### PR DESCRIPTION
**Pull request by Suhail-SSRC**
For some reason, this was in develop at some point of time. Now it isn't anymore!. I am putting this request for it to be added with the latest develop branch update as of 23/05/2022

### CODE REVIEW

File: modules/sc-mesh-secure-deployment/src/gw/main.py [hunk 1]
```diff
@@ -38,15 +38,18 @@ def __init__(self, init_gw_mode: str="server"):
     def clean_and_start(self):
         return_code, null = run_shell_command(
             f"sysctl net.ipv4.ip_forward=1")
+        print("Log 1 for return_code should be !0:" + return_code)
```
Comment (joenpera) [inline modules/sc-mesh-secure-deployment/src/gw/main.py:41]: Lot of print() - calls added even the module uses logger for logging?  Are these really neeeded?

File: modules/sc-mesh-secure-deployment/conf/ap.conf [hunk 2]
```diff
@@ -1,7 +1,4 @@
 network={
-    ssid="AP-TEST"
-    mode=2
-    key_mgmt=WPA-PSK
-    psk="ssrcpassword"
-    frequency=2437
+  ssid="Govind-2G"
```
Comment (martin-tii) [inline modules/sc-mesh-secure-deployment/conf/ap.conf:2]: can you make this generic?

**Review by martin-tii — APPROVED** [review]

**Review by joenpera — CHANGES_REQUESTED** [review]

File: modules/sc-mesh-secure-deployment/src/gw/main.py (hunk 1 above)
Comment (martin-tii) [reply modules/sc-mesh-secure-deployment/src/gw/main.py:41]: Agree, this should go to a log file

Comment (Suhail-SSRC) [reply modules/sc-mesh-secure-deployment/src/gw/main.py:41]: These were for testing purposes, forgot to remove them!

Comment (joenpera) [reply modules/sc-mesh-secure-deployment/src/gw/main.py:41]: Yes, those looks like a debug/testing prints and not needed for the final gw logging.

**Review by govindsi — APPROVED** [review]

**Review by joenpera — APPROVED** [review]

**Comment by joenpera** [comment]
Do we still need these changes?  Please close if this is not valid anymore.

**Comment by martin-tii** [comment]
Not used right now, I'll close it

**Pull request closed without merging**